nixos/modules/services/x11/desktop-managers/xfce4-14.nix

```diff
@@ -69,16 +70,18 @@ in
 
       exo
       garcon
-      gtk-xfce-engine
       libxfce4ui
       xfconf
 
       mousepad
+      parole
+      pavucontrol
```

Comment:
Should this be included only if `config.hardware.pulseaudio.enable`?

Reply:
Yes. I wonder what program non pulseaudio users should use.